src/olympia/abuse/serializers.py

```diff
@@ -257,6 +257,14 @@ def handle_unknown_install_method_or_source(self, data, field_name):
             value = 'other'
         return value
 
+    def handle_addon_signature(self, value):
+        # When the value is starting with `unknown: `, we force the value to
+        # UNKNOWN to account for any unknown value sent by Firefox, see:
+        # https://searchfox.org/mozilla-central/rev/b368ed8b48c0ea8ed2f1948e4776a6fbb5976dff/toolkit/mozapps/extensions/AbuseReporter.sys.mjs#218-219
+        if isinstance(value, str) and value.lower().startswith('unknown: '):
```

Comment:
where is the literal string `unknown: ` coming from and how likely is it to change?

Reply:
Why not save the value anyway? We can always add logic on our end to handle the class of strings tarting with unknown via a model method and this allows us to maintain the real truth of those values.

Reply:
I see now in the code we have similar cases doing the same thing, `other` for example. 👍